terraso_backend/apps/project_management/collaboration_roles.py

```diff
@@ -0,0 +1,18 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+ROLE_MANAGER = "manager"
+ROLE_CONTRIBUTOR = "contributor"
+ROLE_VIEWER = "viewer"
```

Comment:
nit: could this be an enum?

Reply:
I think probably, but I wanted to match the story map implementation for now. We can take another look a bit later if we want.